src/library.js

```diff
@@ -4520,30 +4520,30 @@ LibraryManager.library = {
     }
   },
 
-  emscripten_builtin_mmap2__deps: ['emscripten_with_builtin_malloc', '$syscallMmap2'],
+  emscripten_builtin_mmap2__deps: ['$withBuiltinMalloc', '$syscallMmap2'],
   emscripten_builtin_mmap2: function (addr, len, prot, flags, fd, off) {
-    return _emscripten_with_builtin_malloc(function () {
+    return withBuiltinMalloc(function () {
```

Comment:
It's used without the `$`!? No wonder I couldn't get it to work...

Reply:
Not sure.. i wanted to confirm so I went and made this wider change.